docs/db_overview.md

```diff
@@ -0,0 +1,55 @@
+
+![DB Diagram](assets/db_diagram.png)
+
+The database contains four tables that provide functionality for the program: one primary and three secondary tables. Each table is described as follows:
+
+- **master Table**: This table is the primary table of AEPsych. Important information concerning the experiment is stored here. This table is constantly referenced from within the program for experiment information. The parameters are described as followed: 
+
+
+|Master Table Parameter | Purpose|
+|:-: | :- |
+|unique_id | A unique ID generated for each entry of the database.|
+|experiment_name | A name given to the experiment through the experiment's configuration file. If not specified, default value is set to "default name".|
+|experiment_description | A set of text provided to describe the function of the experiment. This value is configurable in the experiment's configuration file. If not specified, default value is set to "default description".|
+|experiment_id | A value distinguishing the specific experiment in question. This value utilizes the python Universal Unique Identifier module. If not specified in the experiment's configuration file, it will be automatically generated through the python module.|
+|participant_id | A value specifying a particular user within an experiment (perfect for keeping track of more than one participant in an experiment). If not specified in the experiment's configuration file, it will be empty in the Database.|
+| extra_metadata | Any special meta data specified in the configuration file under the section "Metadata". This is automatically generated into a JSON file so it may be serialized/deserialized easier at a later time period. If no metadata is specified in the configuration file, this field will be generated from default values. |
+
+
+- **replay_data Table**: The replay data table is essential for replaying experiments. It contains entries from each step of an experiment's trial so the replay functionality can work properly. The parameters are described as followed:
+
+|Replay Data Parameter | Purpose |
+|:-: | :- |
+|unique_id | A unique ID generated for each entry of the database. |
+|timestamp | An identifier that specifies a timestamp of the specific entry. This parameter helps in providing a chronological order for the replay to replay events. |
+|message_type | This is the type of message generated by the server in this entry. These message types include _setup_, _ask_, and _tell_.|
+|message_contents | Contains the raw data in the message_content field of the message. This is stored as a BLOB in the database.|
```

Comment:
Maybe this is out-of-scope for the current project, but could we store the message as JSON instead of BLOB?